src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write your code here
+const populations = document.getElementsByClassName('population');
+
+const sum = [];
```

Comment:
The variable `sum` is declared as an array, which is appropriate for storing the parsed population numbers. However, the name `sum` might be misleading since it suggests a single total value rather than a collection of values. Consider renaming it to something like `parsedPopulations` for clarity.